engine/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -6,6 +6,7 @@ redirect_from:
 - /engine/articles/dockerfile_best-practices/
 - /docker-cloud/getting-started/intermediate/optimize-dockerfiles/
 - /docker-cloud/tutorials/optimize-dockerfiles/
+- /engine/userguide/eng-image/dockerfile_best-practices/
```

Comment:
Heads up...I think this change caused the best-practices guide to endlessly loop back on itself. :(

https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices.md